src/rgw/setup/rgw.py

```diff
@@ -237,35 +241,35 @@ def _get_rgw_config_path(conf: MappedConf):
         """Return RGW config file path."""
         rgw_config_dir = Rgw._get_rgw_config_dir(conf)
         os.makedirs(rgw_config_dir, exist_ok=True)
-        rgw_conf_file_path = os.path.join(rgw_config_dir, RGW_CONF_FILE)
+        rgw_conf_file_path = os.path.join(rgw_config_dir, const.RGW_CONF_FILE)
         return rgw_conf_file_path
 
     @staticmethod
     def _get_rgw_config_dir(conf: MappedConf):
         """Return RGW config directory path."""
-        config_path = Rgw._get_cortx_conf(conf, CONFIG_PATH_KEY)
-        rgw_config_dir = os.path.join(config_path, COMPONENT_NAME, Rgw._machine_id)
+        config_path = Rgw._get_cortx_conf(conf, const.CONFIG_PATH_KEY)
+        rgw_config_dir = os.path.join(config_path, const.COMPONENT_NAME, Rgw._machine_id)
         return rgw_config_dir
 
     @staticmethod
     def _get_log_dir_path(conf: MappedConf):
         """Return log dir path."""
-        log_path = Rgw._get_cortx_conf(conf, LOG_PATH_KEY)
-        log_dir_path = os.path.join(log_path, COMPONENT_NAME, Rgw._machine_id)
+        log_path = Rgw._get_cortx_conf(conf, const.LOG_PATH_KEY)
+        log_dir_path = os.path.join(log_path, const.COMPONENT_NAME, Rgw._machine_id)
         os.makedirs(log_dir_path, exist_ok=True)
         return log_dir_path
 
     @staticmethod
     def _create_rgw_user(conf: MappedConf):
         """Create RGW admin user."""
-        user_name = Rgw._get_cortx_conf(conf, f'cortx>{COMPONENT_NAME}>auth_user')
-        access_key = Rgw._get_cortx_conf(conf, f'cortx>{COMPONENT_NAME}>auth_admin')
-        auth_secret = Rgw._get_cortx_conf(conf, f'cortx>{COMPONENT_NAME}>auth_secret')
+        user_name = Rgw._get_cortx_conf(conf, f'cortx>{const.COMPONENT_NAME}>auth_user')
+        access_key = Rgw._get_cortx_conf(conf, f'cortx>{const.COMPONENT_NAME}>auth_admin')
+        auth_secret = Rgw._get_cortx_conf(conf, f'cortx>{const.COMPONENT_NAME}>auth_secret')
         err_str = f'user: {user_name} exists'
         # decrypt secret key.
         try:
             cluster_id = Rgw._get_cortx_conf(conf, 'cluster>id')
-            cipher_key = Cipher.gen_key(cluster_id, DECRYPTION_KEY)
+            cipher_key = Cipher.gen_key(cluster_id, const.DECRYPTION_KEY)
             password = Cipher.decrypt(cipher_key, auth_secret.encode('utf-8'))
             password = password.decode('utf-8')
         except CipherInvalidToken as e:
```

Comment:
this change was intended in this PR https://github.com/Seagate/cortx-rgw-integration/pull/44/files#r820619861  

`create_usr_cmd = f'sudo radosgw-admin user create --uid={user_name} --access-key \` 
to
`create_usr_cmd = f'radosgw-admin user create --uid={user_name} --access-key \`

Reply:
done